app/models/cluster_tag.rb

```diff
@@ -1,4 +1,6 @@
 class ClusterTag < ApplicationRecord
   belongs_to :cluster
   belongs_to :tag
+  belongs_to :tenant
```

Comment:
Could we add the `belongs_to` to the `act_as_taggable_on` mixin ?

Reply:
Implemented in https://github.com/RedHatInsights/topological_inventory-core/pull/167